cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/async/progress/OperationProgress.java

```diff
@@ -105,7 +108,8 @@ public synchronized String toString() {
    * @return The map describing the progress of the operation.
    */
   public Map<String, Object> getJsonStructure() {
-    Map<String, Object> operationProgress = new HashMap<>(2);
+    Map<String, Object> operationProgress = new HashMap<>(3);
```

Comment:
I think the original JSON structure is correct, because `OperationProgress`  is one field of `ProgressResult`, we only print the JSON version in top layer of JSON response(here it is `ProgressResult`)